storey/drivers.py

```diff
@@ -70,7 +72,7 @@ class V3ioDriver(NeedsV3ioAccess, Driver):
     :param access_key: V3IO access key. If not set, the V3IO_ACCESS_KEY environment variable will be used.
     """
 
-    def __init__(self, webapi: Optional[str] = None, access_key: Optional[str] = None):
+    def __init__(self, webapi: Optional[str] = None, access_key: Optional[str] = None, use_parallel_operations=True):
```

Comment:
I guess we need an intergration test that actually uses this?

Reply:
this was added to control the feature from mlrun configuration.

Reply:
We use integration tests for that.